spec/acceptance/suites/simp_env/files/target.yaml.example

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
Looking at the SIMP-server's template host YAML file and it raised a few questions. Is one of the nodes supposed to be a SIMP server?  If so, shouldn't it include the `simp::server` class and set simp_options?

Reply:
No, none of the nodes are supposed to be a SIMP server.  The intent is to apply SIMP client settings to all systems, including the bolt-controller.